rosbag2_py/src/rosbag2_py/_transport.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "rosbag2_compression/compression_options.hpp"
+#include "rosbag2_compression/sequential_compression_reader.hpp"
+#include "rosbag2_compression/sequential_compression_writer.hpp"
+#include "rosbag2_cpp/reader.hpp"
+#include "rosbag2_cpp/readers/sequential_reader.hpp"
+#include "rosbag2_cpp/writer.hpp"
+#include "rosbag2_cpp/writers/sequential_writer.hpp"
+#include "rosbag2_storage/storage_options.hpp"
+#include "rosbag2_transport/play_options.hpp"
+#include "rosbag2_transport/rosbag2_transport.hpp"
+
+#include "./pybind11.hpp"
+
+using PlayOptions = rosbag2_transport::PlayOptions;
+using Rosbag2Transport = rosbag2_transport::Rosbag2Transport;
+
+namespace rosbag2_py
+{
+
+class Transport
+{
+public:
+  Transport() {}
+
+  virtual ~Transport() = default;
+
+  void play(
+    const rosbag2_storage::StorageOptions & storage_options,
+    const rosbag2_transport::PlayOptions & play_options)
+  {
+    printf("Entered rosbag2_py::Transport::play\n");
```

Comment:
No common logger used instead of `printf`?

Reply:
Oops! Leftover from development, will remove, good catch